test/framefusion.test.ts

```diff
@@ -63,431 +45,180 @@ describe('framefusion', () => {
-            expect(fileExistsSync('./output/frame-0014.png')).to.be.true;
-            // There are only 13 frames after 150016, so this one should not exist
-            expect(fileExistsSync('./output/frame-0015.png')).to.be.false;
-
-            // Cleanup
-            extractor.dispose();
-        }, 50000);
-
-        it('Should seek and dump frames precisely', async() => {
-            // Arrange
-            const extractor = await BeamcoderExtractor.create({
-                inputFileOrUrl: './test/samples/countTo60.mp4',
-                outputFile: './output/frame-%04d.png',
-                threadCount: 8,
-            }) as BeamcoderExtractor;
-
-            // Act
-            const p1 = performance.now();
-            await extractor.seekToPTS(29696);
-            await extractor.readFrames();
-            const p2 = performance.now();
-            console.log('Time to dump all frames (ms): ', p2 - p1);
-
-            await new Promise(resolve => setTimeout(resolve, 1000));
-
-            // Assert
-            // There should be only 2 frames
-            expect(fileExistsSync('./output/frame-0001.png')).to.be.true;
-            expect(fileExistsSync('./output/frame-0002.png')).to.be.true;
-            expect(fileExistsSync('./output/frame-0003.png')).to.be.false;
-            expect(readFileSync('./output/frame-0001.png')).toMatchImageSnapshot();
-            expect(readFileSync('./output/frame-0002.png')).toMatchImageSnapshot();
-
-            // Cleanup
-            extractor.dispose();
-        }, 50000);
-
-        it('Should dump frames and pause in-between', async() => {
-            // Arrange
-            const extractor = await BeamcoderExtractor.create({
-                inputFileOrUrl: './test/samples/countTo60.mp4',
-                outputFile: './output/frame-%04d.png',
-                threadCount: 8,
-            }) as BeamcoderExtractor;
-
-            // Act
-            // Array generated with
-            // ffprobe -show_frames samples/bbb-smaller.mp4  | grep pts_time | head -n 5 | sed 's/pts_time=//g'
-            await extractor.seekToTime(0);
-
-            const count = 0;
-
-            const onFrameAvailable = async(frame) => {
-                return count < 4;
-            };
-
-            await extractor.readFrames({
-                onFrameAvailable,
-            });
-
-            // Assert
-            expect(readFileSync('./output/frame-0001.png')).toMatchImageSnapshot();
-            expect(readFileSync('./output/frame-0002.png')).toMatchImageSnapshot();
-            expect(readFileSync('./output/frame-0003.png')).toMatchImageSnapshot();
-            expect(readFileSync('./output/frame-0004.png')).toMatchImageSnapshot();
-
-            // Cleanup
-            extractor.dispose();
-        }, 50000);
+        await extractor.dispose();
     });
 
-    describe('Dump frame at time', () => {
-        it.concurrent('Should seek and dump all frames in the video', async() => {
```

Comment:
@animanathome I think we could bring back this test

Reply:
done